api/test/integration/test_agent_POST_endpoints.tavern.yaml

```diff
@@ -1,19 +1,109 @@
 ---
-test_name: POST /agents
+test_name: POST /agents (IPV6)
 
 marks:
   - base_tests
 
 stages:
 
     # POST /agents
-  - name: Try to create a new agent without body
+  - name: Create a new agent with a valid IPV6
     request: &agent_request
       verify: False
       url: "{protocol:s}://{host:s}:{port:d}/agents"
       method: POST
       headers:
         Authorization: "Bearer {test_login_token}"
+      json:
+        name: "NewIPV6Agent"
+        ip: "fdb9:8d4a:1fc2:12a0:11aa:22ff:33eb:99cd"
+    response:
+      status_code: 200
+      json: &agent_create_response
+        error: !anyint
+        data:
+          id: !anystr
+          key: !anystr
+      save:
+        json:
+          agent1_id_ipv6: data.id
+
+    # POST /agents
+  - name: Create a new agent with a compressed IPV6
+    request:
+      verify: False
+      <<: *agent_request
+      json:
+        name: "NewCompressedIPV6Agent"
+        ip: "fdb9::99cd"
+    response:
+      status_code: 200
+      json:
+        <<: *agent_create_response
+      save:
+        json:
+          agent2_id_ipv6: data.id
+
+    # POST /agents
+  - name: Create a new agent with an invalid IPV6
+    request:
+      verify: False
+      <<: *agent_request
+      json:
+        name: "InvalidIPV6Agent"
+        ip: "fdb9:8d4a:1fc2:12a0:11aa:22ff:33eb:99gh"
+    response:
+      status_code: 400
+      json:
+        error: 1706
+
+    # POST /agents
+  - name: Create a new agent with an invalid compressed IPV6
+    request:
+      verify: False
+      <<: *agent_request
+      json:
+        name: "InvalidIPV6Agent"
+        ip: "fdb9::99gh"
+    response:
+      status_code: 400
+      json:
+        error: 1706
+    delay_after: !float "{global_db_delay}"
```

Comment:
I understand that you set the delay here in order to avoid setting 2 delays, don't you? In that case, I think that it is better to put 1 delay after each request creating agents. There are no time problems in this test so there is no need to do this. Moreover, if we happen to delete this test case, we could forget to add a new delay.

Reply:
As we discussed, our best choice is to keep this delay in the last test stage. This way we won't be adding unnecessary delays.

Finally, about the deletion of this stage and the possibility of forgetting to add a new delay: I think the developer of that future request should review this entire test block and completely comprehend every step before deleting it, adding any new delay or value if it's needed.